pkgs/os-specific/darwin/apple-source-releases/configd/default.nix

```diff
@@ -1,11 +1,23 @@
-{ lib, stdenv, appleDerivation', launchd, bootstrap_cmds, xnu, xpc, ppp, IOKit, eap8021x, Security
-, headersOnly ? false }:
-
-appleDerivation' stdenv {
-  meta.broken = stdenv.cc.nativeLibc;
+{ lib
+, appleDerivation'
+, stdenvBootstrap
+, bootstrap_cmds
+, CF
+, launchd
+, ppp
+, xpc
+, IOKit
+, eap8021x
+, Security
+, xnu
+, headersOnly ? false
+}:
+
+appleDerivation' stdenvBootstrap {
+  meta.broken = stdenvBootstrap.cc.nativeLibc;
 
   nativeBuildInputs = lib.optionals (!headersOnly) [ bootstrap_cmds ];
-  buildInputs = lib.optionals (!headersOnly) [ launchd ppp xpc IOKit eap8021x ];
+  buildInputs = lib.optionals (!headersOnly) [ CF launchd ppp xpc IOKit eap8021x ];
```

Comment:
I can add a comment explaining. The bootstrap stdenv doesn’t include CF, libresolv, or Csu. configd needs to use it because it’s a dependency of libresolv, but it also does need CF, so it adds it back as a `buildInput`.